### PR TITLE
Labelingtool

### PR DESCRIPTION
**Pull request by felixbrunner**
The basic version of a labeling tool for image classification to run in a jupyter notebook to accompany a blog article.

The following should be working before the branch is merged:
* installing the tool and its dependencies in a virtual environment following the instructions in the `README`
* running the notebook `run.ipynb` and using the tool inside

### CODE REVIEW

File: labelingtool/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# dida Labeling Tool
+
+This repository contains the code to run a simple labeling tool for image classification in a Jupyter notebook.
+
+<!-- It accompanies [this article](ADD-LINK) in the dida blog. -->
```
Comment (astoff) [inline labelingtool/README.md:5]: Link missing?

File: labelingtool/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Requirements & Installation
+The following are required to run the tool:
+- Python 3
+- IPython
```
Comment (astoff) [inline labelingtool/README.md]: Jupyter already depends on this, no? So we don't need to mention it.

File: labelingtool/README.md (hunk 1 above)
Comment (felixbrunner) [reply labelingtool/README.md:5]: yeah because the article is not posted yet and I didn't have the link yet

Comment (felixbrunner) [reply labelingtool/README.md:5]: I'll add it as soon as the article goes online

File: labelingtool/README.md (hunk 2 above)
Comment (felixbrunner) [reply labelingtool/README.md]: you are right, I will remove it

**Pull request merged**